packages/common/src/utils/system-access-token.ts

```diff
@@ -1,32 +1,31 @@
-import {PropertiesFile} from '../system/files';
+import fse from 'fs-extra';
+import path from 'path';
 
-export function getAccessTokenRCKey(environmentNameOrId: string, dbmsId: string, dbmsUser: string): string {
-    return `//${environmentNameOrId}/${dbmsId}/${dbmsUser}/:_accessToken`;
+import {ENTITY_TYPES} from '../constants';
+
+export function getAccessTokenFileName(environmentNameOrId: string, dbmsId: string, dbmsUser: string): string {
+    return `${environmentNameOrId}-${ENTITY_TYPES.DBMS}-${dbmsId}-${dbmsUser}`;
```

Comment:
We don't actually want names here, right? 

```suggestion
export function getAccessTokenFileName(environmentId: string, dbmsId: string, dbmsUser: string): string {
    return `${environmentId}-${ENTITY_TYPES.DBMS}-${dbmsId}-${dbmsUser}`;
```